src/api/app/models/project.rb

```diff
@@ -100,6 +100,7 @@ def autocomplete(search)
   scope :maintenance_release, -> { where("kind = 'maintenance_release'") }
   scope :home, -> { where("name like 'home:%'") }
   scope :not_home, -> { where.not("name like 'home:%'") }
+  scope :excluded, -> { where.not("name rlike ?", ::Configuration.excluded_projects) if ::Configuration.excluded_projects.present? }
```

Comment:
The name `scope :excluded` seems like it means "projects that are excluded" which isn't what this scope is giving. Maybe call it `scope :filtered_for_index`? Or just `filtered`?